pkg/api/avatar/avatar.go

```diff
@@ -227,9 +309,10 @@ func (a *thunderTask) fetch() error {
 	req.Header.Set("Accept-Language", "zh-CN,zh;q=0.8")
 	req.Header.Set("Cache-Control", "no-cache")
 	req.Header.Set("User-Agent", "Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/33.0.1750.154 Safari/537.36")
+	alog.Debug("Fetching avatar url with parameters", "url", url)
 	resp, err := client.Do(req)
 	if err != nil {
-		a.Avatar.notFound = true
+		av.HandleErrorResponse()
```

Comment:
If an error occurs we return an error. Is it really needed to set the `Avatar` values before returning an error?

Reply:
In this case, yes. The flags we set on the avatar tell the endpoint to return the default "not found" image. We could use default values instead, but I think setting them explicitly every time is safer.